src/appleseed/renderer/modeling/environmentedf/environmentedf.h

```diff
@@ -113,8 +113,13 @@ class APPLESEED_DLLSYMBOL EnvironmentEDF
     virtual float evaluate_pdf(
         const foundation::Vector3f& outgoing) const = 0;        // world space emission direction, unit-length
 
+    virtual bool get_cast_shadows() const;
+
+    virtual void set_cast_shadows(const bool cast_shadows);
+
   protected:
     TransformSequence m_transform_sequence;
+    bool m_env_cast_shadows = true;
```

Comment:
We can't initialize members directly because we still need to support VS 2012 which doesn't support this feature.